scripts/wsrep_sst_clone.sh

```diff
@@ -0,0 +1,1251 @@
+    wsrep_log_debug "-> wait $JOINER_TIMEOUT_WAIT_XST"
+
+    until grep -q ".*<EOF>$" "$WSREP_SST_OPT_DATA/XST_FILE.txt" &> /dev/null
+    do
+         if [ "$JOINER_TIMEOUT_WAIT_XST" == "0" ]; then
+            wsrep_log_error "************ FATAL ERROR ************"
+            wsrep_log_error "TIMEOUT Waiting for the DONOR MESSAGE"
+            wsrep_log_error "Donor message was either incomplete or not sent"
+            donor_message=`cat $WSREP_SST_OPT_DATA/XST_FILE.txt`
+            wsrep_log_error "donor message received: $donor_message"
+            wsrep_log_error "*************************************"
+            exit 1 ;
+         fi
+         sleep 1
+
+        ((JOINER_TIMEOUT_WAIT_XST-=1))
+    done
+
+    wsrep_log_debug "-> WAIT DIR DONOR MESSAGE DONE"
+
+    if ! grep -q "SST@" "$WSREP_SST_OPT_DATA/XST_FILE.txt"; then
+        wsrep_log_info "DONOR SAY IST"
+        wsrep_log_debug "-> RECOVER POSITION TO SEND OUT DONOR IST"
+         RP_PURGED=`cat $WSREP_SST_OPT_DATA/XST_FILE.txt`
+         RP_PURGED=${RP_PURGED%"<EOF>"}
+         wsrep_log_debug "-> POSITION: $RP_PURGED"
+         rm -f  $WSREP_SST_OPT_DATA/XST_FILE.txt || :
+
+         echo $RP_PURGED
+         exit 0
+    else
+         RP_PURGED=`cat $WSREP_SST_OPT_DATA/XST_FILE.txt`
+         wsrep_log_info "DONOR SAY SST ($RP_PURGED)"
+         readonly CLEANUP_CLONE_PLUGIN=${RP_PURGED%|SST*}
+         RP_PURGED_EMERGENCY=${RP_PURGED#*SST@}
+         RP_PURGED_EMERGENCY=${RP_PURGED_EMERGENCY%"<EOF>"}
+         wsrep_log_debug "-> CLONE Plugin to remove? (yes/no) = $CLEANUP_CLONE_PLUGIN"
+         wsrep_log_debug "-> recovered position from DONOR: $RP_PURGED_EMERGENCY"
+    fi
+
+    ##################################################################################################
+    # If we need to SST in any case we must remove the data, so let us do it here and be sure we work on a clean directory
+    wsrep_log_info "Cleaning Data directory $WSREP_SST_OPT_DATA"
+    rm -fr $WSREP_SST_OPT_DATA/*
+
+    # Before starting let us be sure we remove Netcat given it is using same MySQL port
+    # check if there is another netcat process on the IP port we need and try to kill it.
+    NETCAT_KILL=`ps -xo pid,command|grep -e "nc -l -k $JOINER_CLONE_HOST $JOINER_CLONE_PORT"|grep -v "grep"|cut -d ' ' -f 1`
+    if [ ! "$NETCAT_KILL" == "" ];then
+        wsrep_log_info "-> NetCat still up PID: $NETCAT_KILL. Will try to kill it"
+        kill -9 $NETCAT_KILL
+        if [ "$?" != "0" ]; then
+            wsrep_log_debug "-> Cannot kill the existing NetCat process PID $NETCAT_KILL"
+        fi
+    fi
+
+    NC_PID=""
+
+    # No data dir, need to initialize one first, to make connections to
+    # this node possible.
+    # We need to use a temporary empty data directory, because the
+    # actual datadir may already contain some wsrep-related files
+    wsrep_log_info "Initializing data directory at $tmp_datadir"
+
+    wsrep_log_info "INITIALIZE DB"
```

Comment:
We are already printing that we are initializing data directory.
```suggestion
    wsrep_log_info "Initializing data directory at $tmp_datadir"
```

Reply:
removed INITIALIZED DB